src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2TimeoutTests.cs

```diff
@@ -953,7 +953,7 @@ public async Task DATA_Received_SlowlyDueToConnectionFlowControl_DoesNotAbortCon
 
         _mockTimeoutHandler.Verify(h => h.OnTimeout(It.IsAny<TimeoutReason>()), Times.Never);
 
-        AdvanceTime(TimeSpan.FromTicks(1));
+        AdvanceTime(TimeSpan.FromMicroseconds(1));
```

Comment:
nit: FromMicroseconds here vs FromMilliseconds everywhere else.

Why the change from ticks to milliseconds?

Reply:
Reverted. I was having trouble with rounding errors with the mock time provider.